### ISSUE TITLE
super-navbar content showing in the view before being set in the navbar

### ISSUE THREAD

**Issue by chrisnicola** [AppGyver/steroids#882]
While a view is loading (and after loading.html is gone) any content in <super-navbar> just shows as black text in the view before it is added to the navbar. It's pretty awkward but perhaps I'm doing something wrong.

```
  <super-navbar>
    <super-navbar-title ng-bind="navbarTitle"></super-navbar-title>
    <super-navbar-button side="right"></super-navbar-button>
  </super-navbar>
```

**Comment by harsaharsa** [comment]
This should be fixed in Supersonic version 1.1.2, see https://github.com/AppGyver/supersonic/commit/3fe2b618438d7f7c611dbad714b9979c28eb2c41 for the change. Please comment if the issue persists with 1.1.2.

**Comment by chrisnicola** [comment]
I can't seem to get supersonic updated to 1.1.2 using steroids update, or bower update. It says it is installing supersonic#1.1.2, but the folder has a bower.json with version: "1.1.1".

**Comment by chrisnicola** [comment]
Ok this is odd, there is a .bower.json file that has 1.1.2, but a bower.json with 1.1.1

**Comment by chrisnicola** [comment]
So I've tested it with what appears to be 1.1.2 and it is not fixed for me.

**Comment by chrisnicola** [comment]
So I've added:

```
super-navbar-title { display: none; }
```
To my own styles and that fixes this. It actually looks like the commit you link broke this.

**Comment by harsaharsa** [comment]
Right, thanks for reporting – we looked at this on our end and it seems that the 1.1.2 really did mess some things up. Reopening the bug as we investigate the best way to fix this.

**Comment by budda** [comment]
Surprised this bug has been around so long. It's a fundamental thing when testing out the functionality of app building?!
